subcommands/keys/tuf_updates_rotate_offline_key.go

```diff
@@ -185,7 +183,7 @@ func doTufUpdatesRotateOfflineTargetsKey(cmd *cobra.Command) {
 	newProdRoot := genProdTufRoot(newCiRoot)
 
 	fmt.Println("= Re-signing prod targets")
-	newTargetsSigs, err := resignProdTargets(factory, newCiRoot, onlineTargetsId, newCreds)
+	newTargetsSigs, err := resignProdTargets(factory, newKey)
```

Comment:
Will anything bad happen if the `fioctl` process is killed/interrupted at the moment between #179 (key replaced and stored) and #186?
I suppose no, a user will need to run fioctl command again to resign targets with the new targets key?

I wonder whether it's not an issue for the root key rotation when two signatures are required (previous and new root role key signatures).

Reply:
`newKey` is a bit confusing name since it has `*TufSigner` type.

Reply:
> Will anything bad happen if the fioctl process is killed/interrupted ...

There is no chance anything bad can happen until the user runs `fioctl keys tuf apply` command. All these actions are performed upon a temporal metadata, which does not affect devices. So, even if some unknown unknown bug is burrowed here, it only affects that temporal metadata. In any case, at the apply stage, that temporal data is verified for integrity before the commit.

> newKey is a bit confusing name since it has *TufSigner type.

This is how we are used to call this thing in other places like this.
In general, I care little about the correlation between a variable name and a type name. In essence, this variable holds a reference to the new key. A fact that it is wrapped into a convenient intermediary type is secondary here.

Reply:
I though that the key replaced and stored in a file at #179, but it's not so.

Reply:
A new key is stored in a file, correct.
But an old key is _not_ removed from it - it remains in a file for the case if something goes wrong or the user changes their mind.

Reply:
ok, early you said that two keys in a single file is not good :).

Reply:
I got it now. thx.